### ISSUE TITLE
2.1.1 & 2.1.2 Missing intent statements.

### ISSUE THREAD

**Issue by mkhraisha** [ietf-wg-scitt/draft-ietf-scitt-scrapi#18]
The optional API endpoints have intent statements that outline the goal of the Endpoints for example 2.2.1 (Issue Signed Statement Endpoint) has this intent statement:

```
This endpoint enables a Transparency Service to be an issuer of Signed Statements on behalf of authenticated clients. This supports cases where a client lacks the ability to perform complex cryptographic operations, but can be authenticated and report statements and measurements.
```
Such a statement is currently missing from both:
2.1.1 (Transparency Configuration) and 2.1.2 (Signed Statement Registration).

**Comment by JAG-UK** [comment]
I'll take this one and put a PR up with consistency for all the endpoints

**Comment by JAG-UK** [comment]
This was fixed in #28
